web/src/hooks/get-hooks.ts

```diff
@@ -166,25 +168,59 @@ export const useGetUsers = (user: User | null) =>
   });
 
 /**
- * useGetInvites hook returns a query for fetching invites.
+ * useGetProjectInvites hook returns a query for fetching invites.
  *
  * @param {User} user - The user object.
  * @param {string} notebookId - The ID of the notebook.
  * @returns {Query} A query for fetching invites.
  */
-export const useGetInvites = (user: User | null, notebookId: string) =>
+export const useGetProjectInvites = (user: User | null, notebookId: string) =>
   useQuery({
-    queryKey: ['invites', notebookId],
+    queryKey: ['projectinvites', notebookId],
     queryFn: async () => {
-      const invites = await get(`/api/notebooks/${notebookId}/invites`, user);
-
-      for (const invite of invites) {
-        invite.url = `${import.meta.env.VITE_API_URL}/register/${invite._id}`;
-        invite.qrCode = await QRCode.toDataURL(invite.url);
-      }
+      const invites = (await get(
+        `/api/invites/project/${notebookId}`,
+        user
+      )) as GetProjectInvitesResponse;
+      const promises = invites.map(async invite => {
+        const url = `${import.meta.env.VITE_API_URL}/register/${invite._id}`;
+        return {
+          ...invite,
+          url: url,
+          qrCode: await QRCode.toDataURL(url),
+        };
+      });
+      return Promise.all(promises); // Resolving all promises to get enhanced invites
+    },
+    enabled: !!user && !!notebookId, // Only run the query if both user and notebookId are available
+  });
 
-      return invites;
+/**
+ * useGetTeamInvites hook returns a query for fetching invites.
+ *
+ * @param {User} user - The user object.
+ * @param {string} teamId - The ID of the notebook.
+ * @returns {Query} A query for fetching invites.
+ */
+export const useGetTeamInvites = (user: User | null, teamId: string) =>
+  useQuery({
+    queryKey: ['teaminvites', teamId],
+    queryFn: async () => {
+      const invites = (await get(
+        `/api/invites/team/${teamId}`,
+        user
+      )) as GetTeamInvitesResponse;
+      const promises = invites.map(async invite => {
+        const url = `${import.meta.env.VITE_API_URL}/register/${invite._id}`;
```

Comment:
Just noting it here for want of a better spot.  This URL will register someone for a team but what happens afterwards? For project invites we redirect back to the app but here I'm left in old conductor on a page that will soon not exist.  Need to add a redirect back to here.

Reply:
Done - redirects back to the teams page configurably